File: 프로그래머스/lv1/12930. 이상한 문자 만들기/이상한 문자 만들기.js
function solution(s) {
    var answer = '';
    var split_s = s.split(" ")
    for(let i = 0; i < split_s.length; i++){
        for(let j = 0; j < split_s[i].length; j++){
            if(j % 2 == 0){
                answer += split_s[i][j].toUpperCase()
            }else{
                answer += split_s[i][j].toLowerCase()
            }
        }
        if (i < split_s.length -1) {
            answer += ' ';
        }
    }
    return answer;
}

File: 프로그래머스/1/12930. 이상한 문자 만들기/이상한 문자 만들기.py
def solution(s):
    answer = ''
    s_list = s.split(" ")
    for li in s_list:
        for i in range(len(li)):
            if i % 2 == 0:
                answer += li[i].upper()
            else:
                answer += li[i].lower()
        answer += " "
    return answer[:-1]